src/components/core/geoChart/geoChartDataSource.ts

```diff
@@ -44,7 +45,10 @@ function transformPushpinDataSource(geoData: IGeoData): IGeoDataSourceFeatures {
 
             const { lat, lng } = coordinates;
 
-            const sizeValue = hasSize ? sizeData[index] : DEFAULT_PUSHPIN_SIZE_VALUE;
+            const sizeValue = hasSize ? sizeData[index] : pushpinSizeOptionsMap.min.default;
+            const pushpinSize = hasSize
+                ? transformPushpinSizeToPixel(sizeData[index], sizeMin, sizeMax, pushpinSizeOptions)
```

Comment:
the first parameter should be `sizeValue`, right?
I suggest to use `calculatedSizeInPixel` instead of `pushpinSize` here